builtin/providers/cloudstack/resource_cloudstack_instance.go

```diff
@@ -82,6 +82,11 @@ func resourceCloudStackInstance() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"project_name": &schema.Schema{
```

Comment:
Just a small nit, please add an empty line between parameter declarations and I would prefer to keep the name a little shorter to just `project`...